### PR TITLE
docs:  update `wallet init` command line options

### PR DESCRIPTION
**Pull request by qrayven**
# Description of change

Updates options for `wallet init`
Current usage:

```
Usage: wallet init [OPTIONS]

Options:
  -m, --mnemonic-file-path <PATH>  Set the path to a file containing mnemonics. If empty, a mnemonic has to be entered or will be randomly generated
  -n, --node-url <URL>             Set the node to connect to with this wallet [env: NODE_URL=] [default: https://api.testnet.shimmer.network]
  -c, --coin-type <COIN_TYPE>      Coin type, SHIMMER_COIN_TYPE (4219) if not provided [default: 4219]
  -h, --help                       Print help
  -V, --version                    Print version
```

## Links to any relevant issues

#389 

## Type of change

- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/cli/docs/02_wallet.md [hunk 1]
```diff
@@ -81,32 +84,35 @@ When just initialised, the wallet has no account yet, use the `new` command to c
...
-| `--mnemonic` (`-m`)  | Initialises a wallet from an existing mnemonic | ✘                                   | ✘                    |"aunt middle impose ..." |
-| `--node-url` (`-n`)  | Sets the node to connect to                    | https://api.testnet.shimmer.network | `NODE_URL`           | http://localhost:14265  |
-| `--coin-type` (`-c`) | Sets the coin type associated with the wallet  | 4219 (=Shimmer)                     | ✘                    | 4218(=IOTA)             |
+| Name                           | Description                                      | Default                               | Environment Variable | Example                   |
+| ------------------------------ | ------------------------------------------------ |-------------------------------------- |--------------------- | ------------------------- |
+| `--mnemonic-file-path` (`-m`)  | Sets the path to a file containing mnemonics     | ✘                                     | ✘                    | my_mnemonic.txt           |
+| `--node-url` (`-n`)            | Sets the node to connect to                      | <https://api.testnet.shimmer.network> | `NODE_URL`           | <http://localhost:14265>  |
```
Comment (Thoralf-M) [inline documentation/cli/docs/02_wallet.md]: Why the `<>`?

Comment (qrayven) [reply documentation/cli/docs/02_wallet.md]: this is the auto-linter, Should I remove it?

Comment (Thoralf-M) [reply documentation/cli/docs/02_wallet.md]: I would remove it, otherwise one might think one has to also write <> in the cli command which will then fail

Comment (thibault-martinez) [reply documentation/cli/docs/02_wallet.md]: I would remove, yes

Comment (qrayven) [reply documentation/cli/docs/02_wallet.md]: ok, removed

File: documentation/cli/docs/02_wallet.md [hunk 2]
```diff
@@ -81,32 +84,35 @@ When just initialised, the wallet has no account yet, use the `new` command to c
...
 
 Initialize the wallet with a randomly generated mnemonic and the default node.
+
 ```sh
 ./wallet init
 ```
 
-Initialize the wallet with a given mnemonic and the default node.
-DO NOT USE THIS MNEMONIC.
+Initialize the wallet with the default node and a file containing mnemonic.
```
Comment (DaughterOfMars) [inline documentation/cli/docs/02_wallet.md]: `a file containing a mnemonic`

Comment (qrayven) [reply documentation/cli/docs/02_wallet.md]: fixed

**Review by Thoralf-M — DISMISSED** [review]

File: documentation/cli/docs/02_wallet.md [hunk 3]
```diff
@@ -81,32 +84,35 @@ When just initialised, the wallet has no account yet, use the `new` command to c
...
-| -------------------- | ---------------------------------------------- |------------------------------------ |----------------------| ----------------------- |
-| `--mnemonic` (`-m`)  | Initialises a wallet from an existing mnemonic | ✘                                   | ✘                    |"aunt middle impose ..." |
-| `--node-url` (`-n`)  | Sets the node to connect to                    | https://api.testnet.shimmer.network | `NODE_URL`           | http://localhost:14265  |
-| `--coin-type` (`-c`) | Sets the coin type associated with the wallet  | 4219 (=Shimmer)                     | ✘                    | 4218(=IOTA)             |
+| Name                           | Description                                      | Default                             | Environment Variable | Example                   |
+| ------------------------------ | ------------------------------------------------ |------------------------------------ |--------------------- | ------------------------- |
+| `--mnemonic-file-path` (`-m`)  | Sets the path to a file containing mnemonics   	| ✘                                   | ✘                    | my_mnemonic.txt           |
```
Comment (thibault-martinez) [inline documentation/cli/docs/02_wallet.md]: Can you make sure the table is aligned please?

**Review by Thoralf-M — APPROVED** [review]

File: documentation/cli/docs/02_wallet.md (hunk 3 above)
Comment (qrayven) [reply documentation/cli/docs/02_wallet.md]: weird, in vscode it was aligned, the tab happened :)

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**